nautobot/dcim/templates/dcim/interface_edit.html

```diff
@@ -17,6 +17,7 @@
             {% endif %}
             {% render_field form.name %}
             {% render_field form.label %}
+            {% render_field form.status %}
```

Comment:
My preference, at least:

```suggestion
            {% render_field form.status %}
            {% render_field form.label %}
```